src/node/rpc/memberfrontend.h

```diff
@@ -920,13 +920,6 @@ namespace ccf
       members(&network.members)
     {}
 
-    std::vector<uint8_t> get_cert_to_forward(
```

Comment:
This should stay?

Reply:
yes you are right. The very first request needs to be signed, but it is signed by the primary node so that signature does not have much value. What we are doing in Raft is we are skipping verifying said signature and we should do the same thing when running PBFT